core/meta/src/TClass.cxx

```diff
@@ -775,7 +775,7 @@ void TBuildRealData::Inspect(TClass* cl, const char* pname, const char* mname, c
       return;
    }
 
-   Bool_t isTransientMember = kFALSE;
```

Comment:
This change is the cause of the massive failures.

Reply:
This has been replaced :)